shanoir-ng-front/src/app/datasets/dataset-processing/dataset-processing.component.html

```diff
@@ -48,19 +48,19 @@ <h2 class="header command-zone"i18n="Create datasetProcessing|Title@@datasetProc
 					<label>Study</label>
 					<span class="right-col" [ngSwitch]="mode">
 						<ng-template [ngSwitchCase]="'view'">
-                            <a [routerLink]="['/study/details/', study?.id]">
-                                {{study?.name}}
+                            <a [routerLink]="['/study/details/', datasetProcessing?.studyId]">
```

Comment:
don't use intermediate component property when the entity property is sufficient